docs/content/manual/release-specific/v1.1.1/rwx-mount-ownership-reset.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Test RWX share-mount ownership reset
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2357
+
+## Test RWX share-mount ownership
+
+**Given** Setup cluster hosts FQDN.
+```
+root@ip-172-30-0-139:/home/ubuntu# cat /etc/hosts
+127.0.0.1 localhost
+54.255.224.72 ip-172-30-0-139.lan ip-172-30-0-139
+
+root@ip-172-30-0-139:/home/ubuntu# hostname
+ip-172-30-0-139
+
+root@ip-172-30-0-139:/home/ubuntu# hostname -f
+ip-172-30-0-139.lan
+```
+
+*And* `Domain = localdomain` is commented out in `/etc/idmapd.conf` on cluster hosts.
```

Comment:
Do we still need to do this and the following update of `/etc/idmapd.conf` after adding `only_numeric_owners=true`? I thought this is for the workaround which has been documented as a KB.

Reply:
Yes, the /etc/idmapd.conf should not take effect. The test is to ensure the host not already have `Domain = localdomain` setup. If the host already has this pre-set then `localdomain` is enforced as synced between server and client. 
I will make a comment in commit.